zk new broken using group

Feel free to adjust my title. I don't know the best way to describe what is broken here.

I have boiled my setup down to as simple as I can make it. I only have a `.zk` folder in my notebook directory (I removed the one I normally have in `~/.config`). Also for good measure I removed `$ZK_NOTEBOOK_DIR` from my profile.

Quest is my gamification of a journal so its basically the same as the journal example in the docs. Running the following only works if the file for today does not exist. 
```
zk new --group quest Quests
```

If the file exists it fails with the following error, even when I add the `--no-input` flag.
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x88374c]

goroutine 1 [running]:
github.com/alecthomas/kong.catch(0xc000995e88)
        /root/go/pkg/mod/github.com/alecthomas/kong@v0.2.18-0.20210616130643-2d879d20374b/kong.go:398 +0xa5
panic({0x9d9be0, 0xe3f5c0})
        /usr/lib/go/src/runtime/panic.go:1038 +0x215
github.com/mickael-menu/zk/internal/cli/cmd.(*New).Run(0xeaac78, 0xc0001bcf20)
        /home/orlando/zk/internal/cli/cmd/new.go:45 +0x3ac
reflect.Value.call({0x9cd380, 0xeaac78, 0x18}, {0xa543f0, 0x4}, {0xc000422630, 0x1, 0x0})
        /usr/lib/go/src/reflect/value.go:543 +0x814
reflect.Value.Call({0x9cd380, 0xeaac78, 0xb}, {0xc000422630, 0x1, 0x1})
        /usr/lib/go/src/reflect/value.go:339 +0xc5
github.com/alecthomas/kong.callMethod({0xa53c43, 0x3}, {0xa29040, 0xeaac78, 0x3}, {0x9cd380, 0xeaac78, 0x7efc08}, 0xea9f40)
        /root/go/pkg/mod/github.com/alecthomas/kong@v0.2.18-0.20210616130643-2d879d20374b/callbacks.go:71 +0x4e7
github.com/alecthomas/kong.(*Context).RunNode(0xc0003ee900, 0xc000198460, {0xc000995f60, 0x1, 0x1})
        /root/go/pkg/mod/github.com/alecthomas/kong@v0.2.18-0.20210616130643-2d879d20374b/context.go:716 +0x40b
github.com/alecthomas/kong.(*Context).Run(0xc0001bd1e0, {0xc000995f60, 0x1, 0x1})
        /root/go/pkg/mod/github.com/alecthomas/kong@v0.2.18-0.20210616130643-2d879d20374b/context.go:733 +0xa5
main.main()
        /home/orlando/zk/main.go:94 +0x285
```

Here is the configuration I have for that group
```
[group.quest]
paths = ["Quests"]

[group.quest.note]
filename = "{{date now}}"
template = "quest.md"
```

When I changed to tag v0.6.0 using the same config everything works fine so it is something recent.

--- a/internal/cli/cmd/new.go
+++ b/internal/cli/cmd/new.go
@@ -42,15 +42,17 @@ func (cmd *New) Run(container *cli.Container) error {
 		Extra:     cmd.Extra,
 		Date:      time.Now(),
 	})
-	path := filepath.Join(notebook.Path, note.Path)
-	if err != nil {
+	var path string
+	if err == nil {
+		path = filepath.Join(notebook.Path, note.Path)
+	} else {
 		var noteExists core.ErrNoteExists
 		if !errors.As(err, &noteExists) {
 			return err
 		}
 
 		if confirmed, _ := container.Terminal.Confirm(
-			fmt.Sprintf("%s already exists, do you want to edit this note instead?", note.Path),
+			fmt.Sprintf("%s already exists, do you want to edit this note instead?", noteExists.Name),
 			true,
 		); !confirmed {
 			// abort...

